### PR TITLE
chore: bump commander to v9

### LINKED ISSUE #6181: update Commander to 9
## What is the Problem Being Solved?

Our version of https://github.com/tj/commander.js is several versions behind, which makes the documentation trial-and-error. We also don't have access to features like `.conflicts()`.

## Description of the Design

Update to latest release (presently 9.4)

Deal with breaking changes,
- https://github.com/tj/commander.js/releases/tag/v6.0.0
- https://github.com/tj/commander.js/releases/tag/v7.0.0
- https://github.com/tj/commander.js/releases/tag/v8.0.0
- https://github.com/tj/commander.js/releases/tag/v9.0.0

### PR DESCRIPTION
**Pull request by turadg**
closes: #6181

## Description

Upgrade Commander to latest

### Testing Considerations

???

### CODE REVIEW

**Comment by michaelfig** [comment]
Sorry I didn't get to this until now.  Can you please resolve conflicts so the CI runs again, then I'll review?

**Comment by turadg** [comment]
> Can you please resolve conflicts so the CI runs again, then I'll review?

Done. It was just yarn.lock so I don't expect any different CI results

File: packages/agoric-cli/src/main.js [hunk 1]
```diff
@@ -51,8 +51,6 @@ const main = async (progname, rawArgs, powers) => {
     );
   }
 
-  program.storeOptionsAsProperties(false);
```
Comment (michaelfig) [inline packages/agoric-cli/src/main.js:54]: Okay, by reading this file's changes and studying https://www.npmjs.com/package/commander, this looks good.

**Review by michaelfig — APPROVED** [review]
LGTM!  Thanks for the upgrade.

**Pull request merged**